.env.example

```diff
@@ -29,7 +29,12 @@ DB_USERNAME=
 DB_PASSWORD=
 DB_LOG_SQL=false
 
-TIMEZONE=UTC
+# Application timezone. If not specified, the server's default timezone is used.
+# Requires a named timezone identifier.
+# See https://www.php.net/manual/en/timezones.php for the list of supported timezones,
+# except for those listed below "Others".
```

Comment:
```suggestion
# See https://www.php.net/manual/en/timezones.php for the list of supported timezones.
```

The list itself advises against these, so I don't see that we have to. Additionally, UTC remains a valid option.